### ISSUE TITLE
Revisit decisions made regarding usability issues following 2023 audit of GOV.UK Frontend

### ISSUE THREAD

**Issue by kellylee-gds** [alphagov/govuk-design-system#4008]
## What 
There are a few usability issues raised in the 2023 audit of Frontend it would be good to revisit and check we're still happy with our rationale.

 VoiceOver doesn't announce details element's role and states [[https://github.com/alphagov/govuk-design-system/issues/4030](https://www.google.com/url?q=https://github.com/alphagov/govuk-design-system/issues/4030&sa=D&source=calendar&ust=1725368473648054&usg=AOvVaw2Hmc6B5aeIZSrGQVd9Ccly)]

Dragon cannot easily operate the details element [[https://github.com/alphagov/govuk-design-system/issues/4029](https://www.google.com/url?q=https://github.com/alphagov/govuk-design-system/issues/4029&sa=D&source=calendar&ust=1725368473648054&usg=AOvVaw0fNT50wLAAxTIxhQazuxTT)] (which was also found in the website audit)

Dragon cannot easily activate the file upload button [[https://github.com/alphagov/govuk-design-system/issues/4031](https://www.google.com/url?q=https://github.com/alphagov/govuk-design-system/issues/4031&sa=D&source=calendar&ust=1725368473648054&usg=AOvVaw0AH-ZNnwbMHPCDmrQ6zM63)]

## Who needs to work on this
Anika, Owen, beeps, Hazal, Kelly

## Done when
- [x] discussed issues and made agreements
- [x] documented decisions
